src/Resources/config/app/doctrine_migrations.yaml

```diff
@@ -0,0 +1,3 @@
+doctrine_migrations:
+    migrations_paths:
+        'MonsieurBiz\SyliusSettingsPlugin\Migrations': "@MonsieurBizSyliusSettingsPlugin/Migrations"
```

Comment:
Can you use the same method as here instead of this configuration please?

https://github.com/monsieurbiz/SyliusHomepagePlugin/blob/52f4c17a65ab7ba81c95677b7115fddb058ec022/src/DependencyInjection/MonsieurBizSyliusHomepageExtension.php#L45-L53

Also, it needs a configuration, but more like this one:

```
sylius_labs_doctrine_migrations_extra:
    migrations:
        'MonsieurBiz\SyliusSettingsPlugin\Migrations': ~
```